Platform/ArrowlakeBoardPkg/Library/FspsUpdUpdateLib/FspsUpdUpdateLib.c

```diff
@@ -821,6 +827,12 @@ UpdateFspConfig (
       FspsConfig->PchFivrDynPm = 1;
       FspsConfig->D3HotEnable = 0;
       FspsConfig->D3ColdEnable = 1;
+      FspsConfig->ThermalMonitor = 0;
+      FspsConfig->VmdEnable = 0;
+      FspsConfig->C1StateAutoDemotion = 0;
+      FspsConfig->C1StateUnDemotion = 0;
+      FspsConfig->PkgCStateDemotion = 0;
+      FspsConfig->PkgCStateUnDemotion = 0;
```

Comment:
Please check if these UPDs are exposed in config data. use config data to update them if they are exposed.

Reply:
Found out that without these UPDs, s0ix is still able to work.
Removed the changes in the latest commit push